Makefile

```diff
@@ -53,8 +53,8 @@ safety:
 
 ## Run the tests.
 test:
-	docker-compose run --rm -e ENVIRONMENT=testing web \
-		pytest --cov=src/model_warehouse
+	docker-compose run --rm -e ENVIRONMENT=testing -e \
+		POSTGRES_DB_NAME=model_warehouse_test web pytest --cov=src/model_warehouse
```

Comment:
This serves for now but in future this should be done in `settings.py` when the environment is 'testing'.

Reply:
Thanks, I agree with that